robottelo/utils/__init__.py

```diff
@@ -1,14 +1,25 @@
 # General utility functions which does not fit into other util modules OR
 # Independent utility functions that doesnt need separate module
 import base64
+import io
+import json
 import os
 import re
+import time
+import uuid
+import zipfile
 from pathlib import Path
 
+import requests
+from cryptography.hazmat.backends import default_backend
```

Comment:
Looks like we have imported `default_backend as as crypto_default_backend`

Reply:
Updated.